nodes/celestia-app.md

```diff
@@ -49,6 +49,7 @@ Be sure to select the correct network to install the binary for.
 
    ```bash
    make build
```

Comment:
```suggestion
```

Reply:
based on this, going to leave docs as `make install`

https://github.com/celestiaorg/celestia-app/blob/43de42b5a1f83e614da6225e2261bdcf55b1f7a6/Makefile#L24C1-L35